src/commands/alias/set.ts

```diff
@@ -367,7 +369,7 @@ type RemainingAssignAliasErrors = SetDifference<
 function handleCreateAliasError<T>(
   output: Output,
   error: RemainingAssignAliasErrors | T
-): 1 | T {
+): 1 | Error | T {
```

Comment:
This is incorrect IIRC. This function is deliberately implemented to return 1 (when the error is handled) or something else, but not an error. When it returns an error instance it means that there was a known error received that wasn't handled so if you check at the end of the pattern match you'll see the remaining values that should be handled.